liquibase-maven-plugin/pom.xml

```diff
@@ -75,23 +75,42 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-plugin-plugin</artifactId>
-                <version>3.6.4</version>
+                <groupId>org.liquibase.ext</groupId>
+                <artifactId>liquibase-sdk-maven-plugin</artifactId>
+                <version>0-SNAPSHOT</version>
                 <executions>
                     <execution>
-                        <id>generated-helpmojo</id>
+                        <id>default-liquibase-build</id>
+                        <phase>generate-resources</phase>
                         <goals>
-                            <goal>helpmojo</goal>
+                            <goal>build-plugin-xml</goal>
                         </goals>
+                        <configuration>
+                            <liquibaseClassesDir>../liquibase-core/target/classes</liquibaseClassesDir>
+                            <liquibaseProClassesDir>../../liquibase-pro/pro/target/classes</liquibaseProClassesDir>
+                            <outputFile>src/main/resources/META-INF/maven/plugin.xml</outputFile>
```

Comment:
I see you've also committed to the plugin.xml file. Should that be committed or gitignored? Regardless, what's the process for automatically updating that file on a regular basis?

Reply:
In automation this is going to be a problem relying on the pro classes because there are no pro classes when building this code. We maybe even have the liquibase-commercial.jar available currently, since that's just brought in as a special case in building liquibase-dist possibly at the end.